### PR TITLE
Add createPoll function

### PR DESCRIPTION
**Pull request by AstroCB**
Fixed #382 to allow omission of `options` parameter in favor of a callback & to change the format to `optionTitle: selectedState`.

### CODE REVIEW

**Comment by AstroCB** [comment]
@Schmavery

File: src/createPoll.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  return function createPoll(title, threadID, options, callback) {
+    if(!callback) {
+      if(utils.getType(options) == "Function") {
+        callback = options;
+      } else {
+        callback = function() {};
+      }
+    }
+    if(!options) {
+      options = []; // Initial poll options are optional
```
Comment (Schmavery) [inline src/createPoll.js]: This should probably be `{}` now?

**Comment by Schmavery** [comment]
By the way, you can update a pull request by simply pushing more changes to the branch.

**Comment by AstroCB** [comment]
Yep, good catch!

And thanks, I knew there had to be some way, but I couldn't figure it out.

**Comment by Schmavery** [comment]
Thanks @AstroCB !

**Comment by bsansouci** [comment]
Thanks a lot <3

**Pull request merged**